sinatra-active-model-serializers.gemspec

```diff
@@ -13,33 +13,43 @@ Gem::Specification.new do |s|
   s.license = 'MIT'
 
   # generated from git shortlog -sn
-  s.authors = ['saulosantiago']
+  s.authors = ['Saulo Santiago', 'Adriano Tadao']
```

Comment:
@adrianotadao all help deserve to be rewarded in any way. You have been contributor and has improved this code, thanks man :heart: